remotespark/livyclientlib/sqlquery.py

```diff
@@ -0,0 +1,40 @@
+import remotespark.utils.configuration as conf
+
+
+class SQLQuery(object):
+    def __init__(self, query, samplemethod=None, maxrows=None,
+                 samplefraction=None, only_columns=False):
+        if samplemethod is None:
+            samplemethod = conf.default_samplemethod()
+        if maxrows is None:
+            maxrows = conf.default_maxrows()
+        if samplefraction is None:
+            samplefraction = conf.default_samplefraction()
+
+        assert samplemethod == 'take' or samplemethod == 'sample'
+        assert isinstance(maxrows, int)
```

Comment:
![the singularity is coming](https://pbs.twimg.com/profile_images/1788506913/HAL-MC2_400x400.png)